src/sider/oppfolgingsplan/oppfolgingsplaner/OppfolgingsplanerOversikt.tsx

```diff
@@ -89,9 +115,31 @@ export default function OppfolgingsplanerOversikt(props: Props) {
   const hasNoInactivePlans =
     inaktivePlaner.length === 0 && oppfolgingsplanerLPSProcessed.length === 0;
 
+  const activeNarmesteLedere = !!latestOppfolgingstilfelle
+    ? activeNarmesteLederForCurrentOppfolgingstilfelle(
+        currentLedere,
+        latestOppfolgingstilfelle
+      )
+    : [];
+  const activeNarmesteLederIfSingle =
+    activeNarmesteLedere.length === 1 ? activeNarmesteLedere[0] : undefined;
+  const isBeOmOppfolgingsplanEnabled =
+    toggles.isBeOmOppfolgingsplanEnabled &&
+    !!aktivVeileder &&
```

Comment:
Er det noen spesiell grunn til at det sjekkes for `aktivVeileder`?

Reply:
Tenkte bare selv at det ikke skal være mulig å be om oppfølgingsplan om man er som sykmeldt er ufordelt 🤔 Denne antagelsen kan godt være feil. Det er nok uansett ikke en veldig viktig å begrense dette

Reply:
Vi har ingen begrensning at bruker skal være tildelt veilederen i oversikten på annen funksjonalitet i Modia (tilgang til personen holder), så kan ikke helt se noen grunn til at vi skal ha det her :thinking: 
Denne appen har jo egentlig ikke hatt noe forhold til brukers tildelte veileder før vi la på visning og mulighet til å endre den nå i høst.
Uansett så er `aktivVeileder` (fra `useAktivVeilederinfoQuery`) ekstra info om innlogget veileder (navn osv) og ikke brukers tildelte veileder så den kan fjernes :slightly_smiling_face: